app/admin/lists.rb

```diff
@@ -0,0 +1,86 @@
+ActiveAdmin.register List, as: "Wishlist" do
+  actions :all, except: [:create, :new]
+
+  # A list of fields by which the user can filter on the index page
+  filter :title
+  filter :description
+  filter :therapist
+
+  # Each of these scopes appears as a selectable 'tab' on the index page
+  scope :all, default: true
+  scope :empty
+  scope :non_empty
+
+  member_action :delete_requests, method: :delete do
+    begin
+      list = List.find(params[:id])
+      list.gift_requests.destroy_all
+      redirect_to admin_wishlist_path(list), notice: "Gift requests deleted."
+    rescue ActiveRecord::RecordNotFound
```

Comment:
should be changed to a `find_by` nil guard to keep style consistent with other admin actions

i also hate control flow via exceptions, don't know why i did this